core/modules/widgets/checkbox.js

```diff
@@ -3,129 +3,134 @@ title: $:/core/modules/widgets/checkbox.js
 type: application/javascript
 module-type: widget
 
-Checkbox widget
+Checkbox widget to toggle a tag, or set/reset the value of a field or index
 
 \*/
-(function(){
+(function() {
 
 /*jslint node: true, browser: true */
 /*global $tw: false */
 "use strict";
-
 var Widget = require("$:/core/modules/widgets/widget.js").widget;
 
-var CheckboxWidget = function(parseTreeNode,options) {
-	this.initialise(parseTreeNode,options);
-};
-
 /*
 Inherit from the base widget class
 */
+var CheckboxWidget = function(parseTreeNode, options) {
+	this.initialise(parseTreeNode, options);
+};
+
 CheckboxWidget.prototype = new Widget();
 
 /*
 Render this widget into the DOM
 */
-CheckboxWidget.prototype.render = function(parent,nextSibling) {
-	// Save the parent dom node
+CheckboxWidget.prototype.render = function(parent, nextSibling) {
 	this.parentDomNode = parent;
-	// Compute our attributes
 	this.computeAttributes();
-	// Execute our logic
 	this.execute();
-	// Create our elements
 	this.labelDomNode = this.document.createElement("label");
-	this.labelDomNode.setAttribute("class",this.checkboxClass);
+	this.labelDomNode.setAttribute("class", this.checkboxClass);
```

Comment:
Why did you make these edits? It appears the core convention is to not use spaces between parameters. If that was some beautification, I think you'll have to revert it.